src/components/Loader/Loader.tsx

```diff
@@ -40,182 +106,32 @@ export const Loader: FC<Props & ElementProps> = ({
   className = '',
   ...props
 }) => {
-  const theme = useTheme()
-  const classNames = useClassNames()
+  const { wrapper, spinner, line, cog, cogInner, textSlot } = loaderStyle({
+    type,
+    size,
+    className,
+  })
```

Comment:
`wrapper({ className })` してるからここでは不要そう?

```suggestion
  const { wrapper, spinner, line, cog, cogInner, textSlot } = loaderStyle({
    type,
    size,
  })
```

Reply:
https://github.com/kufu/smarthr-ui/pull/3985/commits/ccdd64db4d41aea8da6ffcda18e828515a9d8a02 😎